banking-bench/src/main.rs

```diff
@@ -213,6 +213,12 @@ fn main() {
                 .takes_value(true)
                 .help("Number of threads to use in the banking stage"),
         )
+        .arg(
+            Arg::new("log_messages_bytes_limit")
+                .long("log-messages-bytes-limit")
+                .takes_value(false)
```

Comment:
This seems broken...? How can `log_messages_bytes_limit` end up anything other than None for banking-bench?